UI/window-basic-main.cpp

```diff
@@ -1161,7 +1161,10 @@ void OBSBasic::LoadData(obs_data_t *data, const char *file)
 
 	LogScenes();
 
-	if (obs_missing_files_count(files) > 0) {
+	bool missingFilesShow = config_get_bool(GetGlobalConfig(), "General",
+						"ShowMissingFilesDialog");
+
+	if (obs_missing_files_count(files) > 0 && missingFilesShow) {
```

Comment:
`missingFilesShow` should be moved to the left side of the comparison, otherwise it would still call `obs_missing_files_count` only to discard the result. C++ does not permit reordering the comparison for optimization purposes.